_data/meltano/extractors/tap-google-sheets/matatika.yml

```
@@ -40,6 +40,10 @@ settings:
     Google Sheet.
   label: Child Sheet Name
   name: child_sheet_name
+- description: Optionally set one or more columns from within your Google Sheet to
+    be primary keys for your table. Chosen columns cannot contain null values.
+  label: Key Properties
+  name: key_properties
```

```suggestion
  name: key_properties
  kind: array
```

@DanielPDWalker I popped open https://github.com/Matatika/tap-google-sheets/blob/53edd20eef9a8765e004a1e434247017b2fe9955/tap_google_sheets/tap.py#L48 and noticed that this is an Array type there, should this also be labelled as an array?

---

Yes it should be. Commited this in

---

Thanks :)